develop-up.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! docker info > /dev/null 2>&1; then
+  echo "Ooops - looks like docker is not running"
+  exit 1
+fi
+
+export MDM_PORT=80
+export MDM_APPLICATION_COMMIT=develop
+export MDM_UI_COMMIT=develop
```

Comment:
Setting the correct environment variables seems important. As I was testing on my machine, I was learning about the use of the `.env` file which I think is more useful. If we set these values in the `.env` file, then the `docker compose` commands will automatically pick up those environment variables. 

The `.env` file is the default, but others can be used (I've spotted an irrelevant `.env.cd` file, which I think represents a continuous delivery setup for standard Mauro). The way to use different `.env` files is to pass them in with a command argument:

```bash
docker compose --env-file .env.other ...
```

I think this might help with defining the other environments - Test, Training and Live. I'll make further issues for this.

Action for you @mhocc - could you update the `.env` file please and put these port and commit values into it?

Reply:
Done